.gitignore

```diff
@@ -2,3 +2,4 @@ composer.lock
 Guardfile
 phpunit.xml
 /vendor/
+.idea/
```

Comment:
Personal preferences should not be in the repository.

You can use a global gitignore for this: https://help.github.com/articles/ignoring-files/